Include/cpython/unicodeobject.h

```diff
@@ -99,17 +99,17 @@ typedef struct {
     PyObject_HEAD
     Py_ssize_t length;          /* Number of code points in the string */
     Py_hash_t hash;             /* Hash value; -1 if not set */
+    /* If interned is non-zero, the two references from the
+    dictionary to this object are *not* counted in ob_refcnt.
+    The possible values here are:
+        0: Not Interned
+        1: Interned
+        2: Interned and Immortal
+        3: Interned, Immortal, and Static
+    This categorization allows the runtime to determine the right
+    cleanup mechanism at runtime shutdown. */
+    uint8_t interned;
```

Comment:
I think this remain in the `state` struct. It's okay for a struct to contain both non-bitfield and bitfield members:

* It avoids a potential unnecessary breakage from moving the field
* Keeping it in `state` will make it easier to keep `state` 32-bits due to alignment.